script

```diff
@@ -1,7 +1,23 @@
-sed -i 's/-//g' src/app/shared/swagger/model/Tool.ts && sed -i 's/-//g' src/app/shared/swagger/model/ToolVersion.ts && sed -i 's/-//g' src/app/shared/swagger/model/Metadata.ts 
-rm src/app/shared/swagger/index.ts src/app/shared/swagger/configuration.ts src/app/shared/swagger/variables.ts
-rm -rf src/app/shared/swagger/api
+sed -i 's/-//g' src/app/shared/swagger/model/tool.ts && sed -i 's/-//g' src/app/shared/swagger/model/toolVersion.ts && sed -i 's/-//g' src/app/shared/swagger/model/metadata.ts
+sed -i 's/isPublished/is_published/g' src/app/shared/swagger/model/workflow.ts
```

Comment:
Ya, there's a lot of generated code that's invalid and won't compile.  Styles, I can ignore.